src/main/java/frc/robot/commands/GetCenterNoteX.java

```diff
@@ -52,11 +51,12 @@ public GetCenterNoteX(Translation2d targetNote, DriveTrain driveTrain, NoteVisio
             ( new DeferredCommand(() -> m_driveTrain.followPath(getInitialPath()), Set.of(m_driveTrain))
                 .deadlineWith(new MonitorForNote(noteVision, () -> m_driveTrain.getPose(), m_targetNote, this))
             ).deadlineWith(
-                new WaitUntilCommand(m_driveTrain::isInCenterZone).andThen(new StartIntake(intake, shooter, shooterPivot, elevator))
+                // new WaitUntilCommand(m_driveTrain::isInCenterZone).andThen(new StartIntake(intake, shooter, shooterPivot, elevator))
+                new StartIntake(intake, shooter, shooterPivot, elevator)
```

Comment:
To confirm, this change means we run the Intake the whole time we're driving, correct? To make sure it's fully up to speed by the time we arrive at the note. We were previously waiting until we crossed into center area, but after bumping up the drive speed it seemed that wasn't enough time to fully spin up the Intake.
Related to this-- I think we should discuss/re-examine if we still want 'MonitorForNote'

Reply:
Yes, all agreed.